book/build-svg.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+cd "$(dirname "$0")"
+
+make -j"$(nproc)" svg 
```

Comment:
Throw a -B in here, just so that we always recreate the svgs since as git checks out the tree the .svg output files might end up with a newer timestamp than the .bob input files, which would cause `make` to not generate the .svgs even if they need to be.

Reply:
Added!